tests/commons/test_utils.py

```diff
@@ -39,9 +39,9 @@ def test_files_in_dir(images_path: Path):
     relative_paths = [str(Path(p).relative_to(images_path)) for p in full_paths]
     # THEN all images and their subfolders are in the list
     assert relative_paths == [
-        "sub_dir/img_0.png",
-        "sub_dir/img_1.png",
-        "sub_dir/img_2.png",
+        "sub_dir/img_0.jpeg",
```

Comment:
why did all the formats change?

Reply:
because there is a check for the match between file extension and its contents.